src/httpClient/httpClient.ts

```diff
@@ -150,6 +142,20 @@ export default class HttpClient {
     );
   }
 
+  private static extractRequestLogData(requestConfig?: AxiosRequestConfig): object | undefined {
+    if (!requestConfig) {
+      return undefined;
```

Comment:
Not sure if it will be better to return `{}` or `undefined`. The spread operator `...` works on both as expected. Any opinions?

Reply:
I like `{}` more. As in "empty response"

Reply:
Ok, will adjust it :)